### PR TITLE
Improvements for Ukrainian translation

### PR DESCRIPTION
**Pull request by dvomaks**

### CODE REVIEW

File: app/lang/uk.json [hunk 1]
```diff
@@ -818,61 +819,61 @@
...
+  "Your `Sent Mail` folder could not be automatically detected. Visit Preferences > Folders to choose a Sent folder and then try again.": "Папка `Sent Mail` не може бути автоматично виявлена. Перейдіть \"Налаштування\"> \"Папки\", щоб вибрати папку \"Надіслані\", а потім повторіть спробу.",
+  "Your `Trash` folder could not be automatically detected. Visit Preferences > Folders to choose a Trash folder and then try again.": "Папка \"Trash\" не може бути автоматично виявлена. Перейдіть \"Налаштування\"> \"Папки\", щоб вибрати папку \"Кошик\", а потім повторіть спробу.",
   "Your name": "Твоє ім'я",
-  "Your updated localization will be reviewed and included in a future version of Mailspring.": "Ваша оновлена ​​локалізація буде розглянута і включена в майбутню версію Mailspring.",
-  "Zoom": "Оптимізувати",
+  "Your updated localization will be reviewed and included in a future version of Mailspring.": "Ваша оновлення ​​локалізації будуть розглянуті і включені в майбутню версію Mailspring.",
+  "Zoom": "Збільшити",
   "an email address": "електронна адреса",
-  "an email subject": "тема електронної пошти",
+  "an email subject": "тема електронної листа",
```
Comment (Tester798) [inline app/lang/uk.json]: ^

Comment (dvomaks) [reply app/lang/uk.json]: if you use the word "листа" then right
"тема електронного листа"

**Comment by bengotow** [comment]
Hey folks! Thanks for submitting this, looks like this was a lot of work! @dvomaks is this ready to go? Looks like there might have been a suggestion on there but I'm not sure?

**Comment by bengotow** [comment]
Hey folks! It looks like the review has been addressed in the most recent commit so I'm going to go ahead and merge this in preparation for the release today. If there are other changes feel free to create a new PR! Thanks for all the help, really excited to have a nice Ukranian translation!

**Comment by CodeMouse92** [comment]
Hi @dvomaks and @Tester798, I just wanted to let you know that we've launched the [Mailspring Community](https://community.getmailspring.com). It includes a dedicated section for Translators. We'd love for you to join!

When you do, you can request to be a member of the Translators group here: https://community.getmailspring.com/g/translators

Looking forward to seeing you on there!

-Jason C. McDonald
Volunteer Community Manager
Mailspring

**Pull request merged**